web.py

```diff
@@ -1,16 +1,34 @@
-from flask import Flask, render_template
-
+from cyberbattle import simulation
+from flask import Flask, render_template, request
+from flask.helpers import url_for
+from werkzeug.utils import redirect
 from marlon.simulate import run_simulation
 
 app = Flask(__name__)
 ITERATION_COUNT = 1500
-SIMULATION = run_simulation(ITERATION_COUNT)
+simulation = run_simulation(ITERATION_COUNT, 'tabularq.pkl')
```

Comment:
Can we make it so that the simulation isn't run by default? That way I don't have to wait for tabularq to simulate if I'm just going to immediately decide to simulate deepq or something else.

Bonus feature: Make a command line parameter or environment variable to set it so it does actually run on startup with a specific file as a shortcut for developing so we don't have to upload a file every time. We can save this or both of these for a separate pr if you want.

Reply:
Just changed it to no simulation by default. For environment variable shortcut you're talking about, you just want to change some value between 1 or 2 to differentiate between running either tabularq or deepq?

Reply:
I was thinking just make the environment variable the file to load. So set it to `tabularq.pkl` or whatever and it will load it on startup. If the env var is not set don't load anything on startup.